notebooks/imdb_hotel_conversion.csv

```diff
@@ -0,0 +1,200 @@
+,topic,input,output,label,labeler,description,author,model score
```

Comment:
I think it's fine here for now, but let's keep an eye on creating a subfolder for test trees inside the notebooks directory in the future.

Reply:
In the future I will have plenty of suggestions for handling notebooks :-)